src/components/Avatar/Avatar.tsx

```diff
@@ -80,6 +80,7 @@ export const Avatar = <
           onLoad={() => setLoaded(true)}
           src={image}
           style={{
+            display: 'block',
```

Comment:
Why don't you add this to the `.str-chat__avatar-image` rule? It would be easier for a customer to override it in case they would have to.